cmd/svcat/instance/provision_cmd.go

```diff
@@ -67,6 +68,8 @@ func NewProvisionCmd(cxt *command.Context) *cobra.Command {
 		RunE:    command.RunE(provisionCmd),
 	}
 	command.AddNamespaceFlags(cmd.Flags(), false)
+	cmd.Flags().StringVar(&provisionCmd.externalID, "external-id", "",
```

Comment:
Could you add an example to the section on line 50 also please?

Reply:
Thanks for the review! I just added examples to both provision and bind commands.